Adding the reference to the complete form helpers [ci skip]

ankit8898 opened the pull request:
As the list present below is not updated it makes no sense to update everything here.  Providing the link
for complete reference

ankit8898 committed:
Adding the reference to the complete form helpers
as the list present below is not updated it makes no sense to update everything here.  Providing the link
for complete reference [ci skip]

senny commented on guides/source/form_helpers.md:
```diff
@@ -163,7 +163,9 @@ make it easier for users to click the inputs.
-URL fields, email fields, number fields and range fields:
+URL fields, email fields, number fields and range fields.
+
+Please visit [the Rails Form Helpers documentation](http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html) for complete reference.
```
This sentence seems to interrupt the flow of the guide. Can't we put it at the end of this section after the examples?

ankit8898 committed:
Adding the reference to the complete form helpers
    as the list present below is not updated it makes no sense to update everything here.  Providing the link
    for complete reference [ci skip]

ankit8898 committed:
merged conflicts [ci skip]

ankit8898 commented:
@senny i noticed that above this section the link to API doc is already present. This will become repetitive. Sorry about this one, closing it.
